e2e test invocation script passes even if test fails

This is leading to false positives. See logs of run from https://semaphoreci.com/cdrage/kedge/branches/pull-request-297/builds/6

Script says it passed even if the e2e test failed:

```bash
...
0:30484: dial tcp 192.168.99.100:30484: getsockopt: connection refused
    	e2e_test.go:213: error while making http request "http://192.168.99.100:30484" for service "wordpress:8080", err: Get http://192.168.99.100:30484: dial tcp 192.168.99.100:30484: getsockopt: connection refused
    	e2e_test.go:213: error while making http request "http://192.168.99.100:30484" for service "wordpress:8080", err: Get http://192.168.99.100:30484: dial tcp 192.168.99.100:30484: getsockopt: connection refused
    	e2e_test.go:213: error while making http request "http://192.168.99.100:30484" for service "wordpress:8080", err: Get http://192.168.99.100:30484: dial tcp 192.168.99.100:30484: getsockopt: connection refused
    	e2e_test.go:213: error while making http request "http://192.168.99.100:30484" for service "wordpress:8080", err: Get http://192.168.99.100:30484: dial tcp 192.168.99.100:30484: getsockopt: connection refused
    	e2e_test.go:218: "wordpress:8080" is running!
    	e2e_test.go:451: Successfully pinged all endpoints!
    	e2e_test.go:237: successfully deleted namespace: "customvol"
FAIL
exit status 1
FAIL	github.com/kedgeproject/kedge/tests/e2e	801.607s
envfrom   database-541265016-g35hw   0/1       Terminating   0         59s
envfrom   database-541265016-g35hw   0/1       Terminating   0         59s
./tests/e2e/run_e2e.sh: line 45: kill: (14695) - No such process
```

@surajssd the `customvol` tests have succeeded and have been marked as PASS, while `--- FAIL: Test_Integration/Normal_Wordpress_test` has FAILed. What exactly needs to be investigated?

--- a/scripts/run_e2e.sh
+++ b/scripts/run_e2e.sh
@@ -19,7 +19,7 @@ run_command="go test"
 while getopts ":p:t:" opt; do
     case $opt in
 	p ) PARALLEL=$OPTARG;;
-	r ) TIMEOUT=$OPTARG;;
+	t ) TIMEOUT=$OPTARG;;
     esac
 done
 
@@ -42,4 +42,10 @@ kubectl get po --all-namespaces -w &
 KUBE_PID=$!
 
 # Kill processes once done
-(while kill -0 $TEST_PID; do sleep 5; done) && kill $KUBE_PID
+(while ps -p $TEST_PID > /dev/null; do sleep 5; done) && kill $KUBE_PID
+
+# Get the exit status of the test run
+wait $TEST_PID
+TEST_STATUS=$?
+
+exit $TEST_STATUS

